src/sonic-ctrmgrd/ctrmgr/container_monitor.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import signal
+import sys
+import syslog
+
+sys.path.append('/usr/share/sonic/scripts/')
+import container_state
+
+name = None
+
+def mark_end():
+    container_state.container_down(name)
+    syslog.syslog(syslog.LOG_INFO, "Marking end of docker {}".format(name))
+
+
+def handler(signum, frame):
+    syslog.syslog(syslog.LOG_ERR, "Received signal {}".format(signum))
+    mark_end()
+
+
+def handler(signum, frame):
+    syslog.syslog(syslog.LOG_ERR, "Received signal {}".format(signum))
+    mark_end()
```

Comment:
you have duplicated handler, is it right?

Reply:
Mistake. Normally the duplicate one overrides. I will remove.

Reply:
Removed.